MonoGame.Framework/Graphics/Effect/SpriteEffect.cs

```diff
@@ -1,24 +1,12 @@
-#region File Description
-//-----------------------------------------------------------------------------
-// SpriteEffect.cs
-//
 // Microsoft XNA Community Game Platform
 // Copyright (C) Microsoft Corporation. All rights reserved.
-//-----------------------------------------------------------------------------
```

Comment:
@KonajuGames  - Can/should we add the MonoGame copyright in here too?  We have made lots of modifications past the stock sample.  Does that change the copyright?